modular_skyrat/modules/modular_weapons/code/company_and_or_faction_based/carwo_defense_systems/magazines.dm

```diff
@@ -69,11 +51,8 @@
 /obj/item/ammo_box/magazine/c40sol_rifle/drum
 	name = "\improper Sol rifle drum magazine"
 	desc = "A massive drum magazine for TerraGov rifles, holds sixty rounds."
-
 	icon_state = "rifle_drum"
-
-	w_class = WEIGHT_CLASS_BULKY
-
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
yea uhh the drums are kind of horrific, this is really my main concern, don't really want people to have multiple magazines of 60 shots

Reply:
gonna just axe crew availability of the drum mags